src/functionalTest/java/uk/gov/hmcts/reform/feature/FeatureToggleTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.feature;
+
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import uk.gov.hmcts.reform.feature.categories.SmokeTestCategory;
+
+import java.io.IOException;
+import java.util.UUID;
+
+public class FeatureToggleTest extends BaseTest {
```

Comment:
should that be `UpdateFeatureToggleTest`? (the other test class is called `CreateFeatureToggleTest`)

Reply:
no it's enabling/disabling feature toggle not updating the properties of feature toggle.

Reply:
'change feature toggle state' test then?

Reply:
or 'switch feature toggle' test. ATM you've basically got `CreateFooBarTest` and `FooBarTest` which is confusing